.changeset/breezy-days-prove.md

```diff
@@ -0,0 +1,5 @@
+---
+'@lit-labs/motion': patch
```

Comment:
We shouldn't do a release for a test fix, so this should be an empty change set.

Reply:
Done.